app/controllers/api/v2/roles_controller.rb

```diff
@@ -23,6 +23,8 @@ def show
       def_param_group :role do
         param :role, Hash, :required => true, :action_aware => true do
           param :name, String, :required => true
+          param :organization_ids, Array
```

Comment:
`param_group :taxonomies, ::Api::V2::BaseController` for consistency with the rest of the API